pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,274 @@
+
+# Bento in action
+
+The example below demonstrates how to include `amp-base-carousel` and `amp-accordion` in a non-AMP HTML page.
+
+```
+<!DOCTYPE html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8" />
+    <title>AMP Public Radio</title>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script
+      async
+      custom-element="amp-base-carousel"
+      src="https://cdn.ampproject.org/v0/amp-base-carousel-1.0.js"
+    ></script>
+    <script async custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-1.0.js"></script>
+    <script>
+      ((self.AMP = self.AMP || []).push(function (AMP) {
+        AMP.toggleExperiment('bento', true);
+      }));
+    </script>
+    <style>
+      amp-base-carousel {
+        display: block;
+        height: 400px;
+        width: 600px;
+        overflow: hidden;
+      }
+      amp-accordion {
+        width: 600px;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Bento AMP Components</h1>
+      <p>
+        This demo includes the following components and configurations:
+        <ul>
+          <li>`amp-base-carousel` with looping and autoadvancing on 5s interval</li>
+          <li>`amp-accordion` with three expandable sections</li>
+        </ul>
+      </p>
+      <amp-base-carousel
+        id="my-carousel"
+        class="carousel-style"
+        controls="never"
+        loop
+        auto-advance
+        auto-advance-interval="5000"
+      >
+      <img
+          class="article-img"
+          width="1024"
+          height="682"
+          src="https://raw.githubusercontent.com/ampproject/amp.dev/future/examples/static/samples/img/green_apple_1_1024x682.jpg"
+        >
+      <img
+          class="article-img"
+          width="1024"
+          height="682"
+          src="https://raw.githubusercontent.com/ampproject/amp.dev/future/examples/static/samples/img/golden_apple1_1024x682.jpg"
+        >
+        <img
+            class="article-img"
+            width="1024"
+            height="682"
+            src="https://raw.githubusercontent.com/ampproject/amp.dev/future/examples/static/samples/img/product2_1024x682.jpg"
+          >
+      </amp-base-carousel>
+    <button id="previous-button">
+      Previous Slide
+    </button>
+    <button id="next-button">
+      Next Slide
+    </button>
+    <script>
+        const nextButton = document.querySelector('#next-button');
+        const prevButton = document.querySelector('#previous-button');  
+        const carousel = document.querySelector('amp-base-carousel');
+        
+        customElements
+        .whenDefined('amp-base-carousel')
+        .then(() => {
+            return carousel.getApi();
+        })
+        .then((api) => {
+            nextButton.addEventListener('click', () => api.next());
+            prevButton.addEventListener('click', () => api.prev());
+        });
+    </script>
+    <amp-accordion id="my-accordion" disable-session-states>
```

Comment:
TBH, I think we don't need the accordion here. Demonstrating a single component should be sufficient to show how bento works. I think the carousel example works great for this.